internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -896,3 +931,39 @@ func flattenKustoClusterLanguageExtensionList(extensions *clusters.LanguageExten
 
 	return output
 }
+
+func expandKustoClusterCalloutPolicies(input []interface{}) *[]clusters.CalloutPolicy {
+	if len(input) == 0 {
+		return nil
+	}
+
+	policies := make([]clusters.CalloutPolicy, 0)
+	for _, item := range input {
+		policyMap := item.(map[string]interface{})
+		policy := clusters.CalloutPolicy{
+			CalloutType:     pointer.To(clusters.CalloutType(policyMap["callout_type"].(string))),
```

Comment:
Arguably, we shall use `pointer.ToEnum()` instead (though I don't see any benefit personally...).